SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialAuthorityTest/SpatialAuthorityTest.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SpatialFunctionalTest.h"
+#include "SpatialAuthorityTest.generated.h"
+
+class ASpatialAuthorityTestActor;
+class ASpatialAuthorityTestReplicatedActor;
+
+/** Check SpatialAuthorityTest.cpp for Test explanation. */
+UCLASS()
+class SPATIALGDKFUNCTIONALTESTS_API ASpatialAuthorityTest : public ASpatialFunctionalTest
+{
+	GENERATED_BODY()
+
+public:
+	ASpatialAuthorityTest();
+
+	virtual void BeginPlay() override;
+
+	virtual void GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const override;
+
+	virtual void FinishStep() override { ResetTimer(); Super::FinishStep();	};
+
+	void ResetTimer() {	Timer = 0.5; };
```

Comment:
```suggestion
	void ResetTimer() { Timer = 0.5; };
```
?

Reply:
I don't know man, since clang I think things started to be very odd with spacing. Here for some reason it aligned Timer with a tab, but I can only see it when I show whitespaces:
![image](https://user-images.githubusercontent.com/5429121/91145789-4c7e3f80-e6b6-11ea-988a-6a85933384c1.png)